### ISSUE TITLE
kl_div loss doc reference link goes nowhere

### ISSUE THREAD

**Issue by zou3519** [pytorch/pytorch#27852]
## 📚 Documentation

https://pytorch.org/docs/master/nn.functional.html#torch.nn.functional.kl_div

![image](https://user-images.githubusercontent.com/5652049/66766584-6804a100-ee7c-11e9-9c32-fb9dd7cde5b8.png)

**Comment by thomasjpfan** [comment]
Closing because this was fixed in https://github.com/pytorch/pytorch/pull/37778.

https://github.com/pytorch/pytorch/blob/9d5617603447297ccf730c19736f44c8dae045ab/torch/nn/functional.py#L2669-L2670
